web/client/components/geostory/contents/WebPageWrapper.jsx

```diff
@@ -0,0 +1,101 @@
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import Dialog from '../../misc/StandardDialog';
+import { FormControl, ControlLabel, FormGroup, Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { setWebPageUrl } from '../../../actions/geostory';
+import Message from '../../I18N/Message';
+import Portal from '../../misc/Portal';
+import PropTypes from 'prop-types';
+import WebPage from './WebPage';
+import { compose, withHandlers } from 'recompose';
+import { isValidURL } from '../../../utils/URLUtils';
+
+export class WebPageWrapper extends React.PureComponent {
+    static propTypes = {
+        editURL: PropTypes.bool,
+        onClose: PropTypes.func,
+        onChange: PropTypes.func,
+        src: PropTypes.string,
+        id: PropTypes.string
+    }
+
+    state = {
+        url: this.props.src || '',
+        error: false
+    }
+
+    render() {
+        const { onClose, editURL } = this.props;
+        const { url, error } = this.state;
+        return (
+            <React.Fragment>
+                <Portal>
+                    <Dialog
+                        modal
+                        show={editURL}
+                        onClickOut={(e) => e.preventDefault() && e.stopPropagation()}
+                        title={<Message msgId="geostory.webPageCreator.title" />}
+                        onClose={onClose}
+                        id="web-page-creator"
+                        footer={(
+                            <Button bsSize="small" onClick={this.save} >
+                                <Message msgId="geostory.webPageCreator.saveButton" />
+                            </Button>
+                        )}
+                    >
+                        { error && (
+                            <div className="dropzone-errorBox alert-danger">
+                                <Message msgId="geostory.webPageCreator.error"/>
+                            </div>
+                        )}
+                        <FormGroup controlId="WEBPAGE_URL" validationState={error && 'error'}>
+                            <ControlLabel>
+                                <Message msgId="geostory.webPageCreator.url.label" />
+                            </ControlLabel>
+                            <FormControl
+                                label="URL"
+                                type="text"
+                                value={url}
+                                onChange={this.updateURL}
+                                required
+                            />
+                        </FormGroup>
+                    </Dialog>
+                </Portal>
+                <WebPage {...this.props} />
+            </React.Fragment>
+        );
+    }
+
+    updateURL = ({ target: { value: url }}) => {
+        this.setState({ url });
+    }
+
+    save = () => {
+        const { url } = this.state;
+        const error = !isValidURL(url);
```

Comment:
can you pass ConfigUtils.getConfigProp("GeoStoryValidIframeURLRegex") as second argument? 
in that case we can override it anytime without changing the code

Verify that if the property is missing in config, then it uses the default one

Reply:
ok

Reply:
@MV88 done